packages/@o3r/configuration/src/devkit/configuration-devtools.message.service.ts

```diff
@@ -78,8 +78,7 @@ export class ConfigurationDevtoolsMessageService implements OnDestroy, DevtoolsS
 
   /**
    * Function to handle the incoming messages from Otter Chrome DevTools extension
-   * @param event Event coming from the Otter Chrome DevTools extension
-   * @param message
+   * @param message message coming from the Otter Chrome DevTools extension
```

Comment:
I think it's more readable if the description always starts with a capital letter

Reply:
We can discuss to add this rule [jsdoc/require-description-complete-sentence](https://github.com/gajus/eslint-plugin-jsdoc/blob/main/.README/rules/require-description-complete-sentence.md)

Reply:
As discussed if we go for this rule we will have a dedicated PR to activate it